sandworm/lib/web.mlx

```diff
@@ -125,7 +125,7 @@ $ dune exec hello_world
 
 
 let q_and_a = [
-    "Is it stable?", (<p>"No. We are working to stabilize these features and move them from a Developer Preview to a regular released version of Dune. In the meantime, expect some bugs and please report them to "<Link href="https://github.com/ocaml/dune/issues">"ocaml/dune"</Link></p>)
+    "Is it stable?", (<p>"No. For a stable release of OCaml please follow the installation guide on "<Link href="https://ocaml.org">OCaml.org</Link>". The Developer Preview is an unstable distribution of Dune, and by nature will always be unstable. However, its features will stabilize over time and move to a stable release of Dune. If you use the Developer Preview expect bugs and please report them to "<Link href="https://github.com/ocaml/dune/issues">"ocaml/dune"</Link></p>)
```

Comment:
```suggestion
    "Is it stable?", (<p>"No. For a stable release of OCaml, please follow the installation guide on "<Link href="https://ocaml.org">OCaml.org</Link>". The Developer Preview is an unstable distribution of Dune and, by nature, will always be unstable. However, its features will stabilise over time and move to a stable release of Dune. If you use the Developer Preview, expect bugs and please report them to "<Link href="https://github.com/ocaml/dune/issues">"ocaml/dune"</Link></p>)
```

Minor grammar, etc. I can fix these things without troubling you if you wantn to give me write access. 🙂